composer.json

```diff
@@ -13,21 +13,21 @@
     "require": {
         "php": "^5.6|^7.0",
         "egeloen/json-builder": "^2.0|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/form": "^2.7|^3.0",
-        "symfony/framework-bundle": "^2.7|^3.0"
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
+        "symfony/form": "^2.7|^3.0|^4.0",
+        "symfony/framework-bundle": "^2.7|^3.0|^4.0"
     },
     "require-dev": {
         "composer/composer": "^1.0",
         "friendsofphp/php-cs-fixer": "^2.0",
         "phpunit/phpunit": "^5.0|^6.0",
         "sensio/distribution-bundle": "^3.0.12|^4.0|^5.0",
-        "symfony/asset": "^2.7|^3.0",
-        "symfony/console": "^2.7|^3.0",
-        "symfony/phpunit-bridge": "^2.7|^3.0",
-        "symfony/templating": "^2.7|^3.0",
-        "symfony/twig-bridge": "^2.7|^3.0",
-        "symfony/yaml": "^2.7|^3.0",
+        "symfony/asset": "^2.7|^3.0|^4.0",
```

Comment:
Symfony 3.2 support ends in few days so you can use `^3.3` here.

Reply:
Not the point of this PR. I do not want to end up changing more than I have to, SF4 support is delayed long enough as it is.

Reply:
@kunicmarko20 i believe is 3.3 that ends support so would be 3.4. But yeah nitpicks